lib/jekyll/drops/static_file_drop.rb

```diff
@@ -0,0 +1,11 @@
+module Jekyll
+  module Drops
+    class StaticFileDrop < Drop
+      extend Forwardable
+      def_delegators :@obj, :name, :extname, :modified_time, :basename
+      def_delegator :@obj, :relative_path, :path
```

Comment:
This may not be 100% the same as below. We have to ensure this has the forward slash at the start!